CHANGELOG.md

```diff
@@ -15,6 +15,16 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * The minimum supported Rust version (MSRV) is now 1.84.0.
 * The `git.push-branch-prefix` config has been removed in favor of
   `git.push-bookmark-prefix`.
+* `jj unsquash` has been removed in favor of `jj squash` and
```

Comment:
nit: can you add blank lines between the bullets for consistency with the rest of this file?